stable/sematext-agent/README.md

```diff
@@ -1,35 +1,61 @@
 # Sematext Agent
 
-Sematext Agent collects metrics about hosts (CPU, memory, disk, network, processes), containers (both Docker and rkt) and orchestrator platforms and ships that to [Sematext Cloud](https://sematext.com/cloud). Sematext Cloud is available in the US and EU regions.
+Sematext Agent collects logs and monitoring hosts (CPU, memory, disk, network, processes, ...), containers and orchestrator platforms and ships that to [Sematext Cloud](https://sematext.com/cloud). Sematext Cloud is available in the US and EU regions.
 
 ## Introduction
 
 This chart installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
 
 ## Prerequisites
 
-- Kubernetes 1.9+
+- Kubernetes 1.13+
 - You need to create [a new Docker app in Sematext Cloud](https://apps.sematext.com/ui/integrations/create/docker) to get relevant tokens
 
 ## Installation
 
-To install the chart run the following command:
+To install the chart for logs run the following command:
 
 ```bash
-$ helm install --name release_name \
-    --set containerToken=YOUR_CONTAINER_TOKEN,logsToken=YOUR_LOGS_TOKEN stable/sematext-agent
+$ helm install sematext-logagent  \
+    --set logsToken=YOUR_LOGS_TOKEN \
+    stable/sematext-agent
+```
+
+To install the chart for monitoring run the following command:
+
+```bash
+$ helm install sematext-agent \
+    --set containerToken=YOUR_CONTAINER_TOKEN \
+    --set infraToken=YOUR_INFRA_TOKEN \
```

Comment:
Mention the region param too? You can show it with default value `region=US` and add a note that mentions value EU in case SC EU is used. And the same thing for logs and logs+monitoring variants below and above.